Readme.md

```diff
@@ -3,20 +3,20 @@
 [![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/E424)
 [![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
 <!-- default badges end -->
-<!-- default file list -->
-*Files to look at*:
 
-* **[Form1.cs](./CS/DropDownButtonExample/Form1.cs) (VB: [Form1.vb](./VB/DropDownButtonExample/Form1.vb))**
-* [Form1.Designer.cs](./CS/DropDownButtonExample/Form1.Designer.cs) (VB: [Form1.Designer.vb](./VB/DropDownButtonExample/Form1.Designer.vb))
-<!-- default file list end -->
-# How to use the DropDownButton control
+# PopupMenu for Windows Forms - How to show a PopupMenu as a DropDownButton control's dropdown
 
+The [DropDownButton](https://docs.devexpress.com/WindowsForms/DevExpress.XtraEditors.DropDownButton) control is a button with an associated popup control. This example demonstrates how to assign a [PopupMenu](https://docs.devexpress.com/WindowsForms/DevExpress.XtraBars.PopupMenu) to a [DropDownButton](https://docs.devexpress.com/WindowsForms/DevExpress.XtraEditors.DropDownButton) control at design time and runtime. 
+The [DropDownButton.DropDownControl](https://docs.devexpress.com/WindowsForms/DevExpress.XtraEditors.DropDownButton.DropDownControl) property is used for this purpose.
```

Comment:
Is it possible to combine this sentence with the preceding sentence (and switch the sentence to the active voice as well)? For example, is the following technically correct?

`This example uses the [DropDownButton.DropDownControl](https://docs.devexpress.com/WindowsForms/DevExpress.XtraEditors.DropDownButton.DropDownControl) property to assign a [PopupMenu](https://docs.devexpress.com/WindowsForms/DevExpress.XtraBars.PopupMenu) to a [DropDownButton](https://docs.devexpress.com/WindowsForms/DevExpress.XtraEditors.DropDownButton) control at design time and runtime.`

Reply:
```suggestion
This example uses the [DropDownButton.DropDownControl](https://docs.devexpress.com/WindowsForms/DevExpress.XtraEditors.DropDownButton.DropDownControl) property to assign a [PopupMenu](https://docs.devexpress.com/WindowsForms/DevExpress.XtraBars.PopupMenu) to a [DropDownButton](https://docs.devexpress.com/WindowsForms/DevExpress.XtraEditors.DropDownButton) control at design time and runtime.
```